docs/management/custom_proposals.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 6
+sidebar_label: Custom Function Call
```

Comment:
Let's call this "Function Calls" for consistency with the UI. Can we please nest this under a new "Advanced" section (Damian can also move this later).